### ISSUE TITLE
[Feature] Test/Train Splitter

### ISSUE THREAD

**Issue by pablomendes** [dbpedia-spotlight/dbpedia-spotlight#52]
Converted from [SourceForge issue 3520352](http://sourceforge.net/support/tracker.php?aid=3520352), submitted by pablomendes

Create a class that takes the TSV file generated by ExtractOccsFromWikipedia and extracts a sample of X% of the occurrences as test set, and keeps the rest  (100-X%) as training set. Things to consider: 
- sorting by URI is probably a precondition
- we do not want to get unambiguous URIs into the test set. In fact, a minimum count "threshold" could be set so that we'd ignore URIs under that threshold both from test and from training sets.
